test_requirements.txt

```diff
@@ -1,5 +1,5 @@
-pytest
-openapiart.egg
+flask
 openapiart
-Flask
-sanity
+pytest
+flake8
+black==22.1.0 ; python_version > '2.7'
```

Comment:
why should black and openapiart be in test_requirements.txt ?

also when should do.py install test_requirements.txt ?